assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -171,6 +171,10 @@
 
   .component-feature-list__item {
     margin-bottom: $gu-v-spacing / 2;
+
+    @include mq($from: leftCol) {
+      width: 200px;
+    }
```

Comment:
Is there a better way for me to avoid ⬇️ 
![image](https://user-images.githubusercontent.com/3072877/45756846-da736500-bc19-11e8-82e5-a5549080dd88.png)

than setting a width on the item or maybe the text at this breakpoint?